corehq/apps/userreports/pillow.py

```diff
@@ -57,9 +57,18 @@ def rebuild_table(self, table):
         table.rebuild_table()
 
     def python_filter(self, doc):
-        # filtering is done manually per indicator set change_transport
+        # filtering is done manually per indicator see change_transport
         return True
 
+    def change_trigger(self, changes_dict):
```

Comment:
just curious as to why this wasn't working in the first place. it seems when `process_change` is called both `change_trigger` and `change_transport` are called (https://github.com/dimagi/pillowtop/blob/master/pillowtop/listener.py#L289) unless `change_trigger` or `change_transform` is `None` which doesn't seem like that would happen. `change_transport` then should have deleted the doc from the table. one of these deleting mechanisms appears redundant to me